### PR TITLE
fix: Precede preCommand and postCommand with `set -e`

### PR DESCRIPTION
**Pull request by deutschj**
The templated `preCommand` and `postCommand` values should be preceded with `set -e`, so that the Renovate Kubernetes CronJob fails directly on every non-zero exit code.

### CODE REVIEW

File: charts/renovate/templates/cronjob.yaml [hunk 1]
```diff
@@ -84,10 +84,12 @@ spec:
               args:
                 - |
                 {{- if .Values.cronjob.preCommand }}
+                  set -e
```
Comment (viceice) [inline charts/renovate/templates/cronjob.yaml:87]:
```suggestion
```

File: charts/renovate/templates/cronjob.yaml [hunk 2]
```diff
@@ -84,10 +84,12 @@ spec:
               args:
                 - |
```
Comment (viceice) [inline charts/renovate/templates/cronjob.yaml:85]:
```suggestion
                - |
                  set -e;
```

File: charts/renovate/templates/cronjob.yaml [hunk 3]
```diff
@@ -84,10 +84,12 @@ spec:
...
                 {{- if .Values.cronjob.preCommand }}
+                  set -e
                   {{- .Values.cronjob.preCommand | nindent 18 }}
                 {{- end }}
                   renovate
                 {{- if .Values.cronjob.postCommand }}
+                  set -e
```
Comment (viceice) [inline charts/renovate/templates/cronjob.yaml:92]:
```suggestion
```

File: charts/renovate/templates/cronjob.yaml [hunk 4]
```diff
@@ -83,6 +83,7 @@ spec:
               command: ["/bin/bash", "-c"]
               args:
                 - |
+                  set -e;
```
Comment (viceice) [inline charts/renovate/templates/cronjob.yaml:86]: Is the indention right? did you tested it?

**Comment by viceice** [comment]
Please use the re-request review button next to my reviewer name

**Comment by viceice** [comment]
please add a test

File: charts/renovate/templates/cronjob.yaml (hunk 4 above)
Comment (deutschj) [reply charts/renovate/templates/cronjob.yaml:86]: Yes, the indentation is correct, I've tested it.

**Comment by deutschj** [comment]
> please add a test

what kind of tests should i add exactly?

**Comment by viceice** [comment]
> > please add a test
> 
> what kind of tests should i add exactly?

Change one test

**Review by viceice — APPROVED** [review]

**Pull request merged**